### PR TITLE
Import KeyCodeCombinations enums to related classes #916

### LINKED ISSUE #916: Import KeyCodeCombinations enums to related classes
Discussed in #909 

In ```ListPanel```, ```PanelControl```, ```NavigableListView```, ```MenuControl```, ```RepositorySelector``` and a few tests.

### PR DESCRIPTION
**Pull request by HyungJon**
Fixes #916

### CODE REVIEW

File: src/main/java/ui/listpanel/ListPanel.java [hunk 1]
```diff
@@ -145,138 +172,137 @@ private void setupListView() {
...
-            if (KeyboardShortcuts.SWITCH_DEFAULT_REPO.match(event)) {
+            if (SWITCH_DEFAULT_REPO.match(event)) {
                 ui.switchDefaultRepo();
             }
-            if (KeyboardShortcuts.UNDO_LABEL_CHANGES.match(event)) {
+            if (UNDO_LABEL_CHANGES.match(event)) {
                 ui.triggerNotificationAction();
             }
-            for (KeyCodeCombination key:KeyboardShortcuts.JUMP_TO_FIRST_ISSUE_KEYS) {
+            for (KeyCodeCombination key:JUMP_TO_FIRST_ISSUE_KEYS) {
```
Comment (dariusf) [inline src/main/java/ui/listpanel/ListPanel.java]: This should be spaced out properly.

**Comment by dariusf** [comment]
Hmm, maybe we shouldn't static import those that are not NAMED_LIKE_CONSTANTS... May be confusing in the contexts in which they are used.

**Comment by HyungJon** [comment]
It does make sense since those aren't immutable, we might not want to import them like that. Making the change.

**Comment by dariusf** [comment]
Going to assume the coverage drop is a false positive. Could you squash? I'll merge then.

**Comment by HyungJon** [comment]
Okay. Can advice me on how to squash when there's a merge (with all involved commits) in between?

**Comment by dariusf** [comment]
`git rebase -i origin/master` should do it.

Hmm, it's more involved than I thought. I managed to get it on my end, but if you 
git checkout Issue916


```sh
git rebase -i origin/master
# Fix merge conflicts
git add .
git rebase --continue

# If everything fails,

```

**Comment by HyungJon** [comment]
Right. That should be squashed now.

**Comment by dariusf** [comment]
Nice.

**Pull request merged**